generative_data_prep/data_prep/pipeline.py

```diff
@@ -122,6 +122,11 @@ def rename_files(
             os.rename(os.path.join(split_dir, new_name), os.path.join(test_dir, new_name))
         else:
             files_to_tokenize.append(new_name)
+        os.path.getsize(new_file_path)
+    if not os.path.getsize(new_file_path) > 0:
+        raise ValueError(
+            "The number of total splits exceeds the number of data entries. Please reduce the number of splits."
+        )
```

Comment:
I like that you add this check in as well as the other check, I did not think of this. Looks great!

Reply:
nit: update message from  
"The number of total splits exceeds the number of data entries. Please reduce the number of splits." 
to 
"The number of total splits exceeds the number of lines in the input path jsonl file. Please reduce the number of splits, or increase the number of lines in the dataset."